### DISCUSSION TITLE
2.2.3 Rocky / RHEL packages

### DISCUSSION THREAD

**Discussion by hanneskasp** [openzfs/zfs#15937] [General]
Hello,
am I doing something wrong, because I don't get zfs 2.2.3 packages on my Rocky system?

I enabled the testing branch and I have 2.2.2 since some weeks. Now I like to upgrade to 2.2.3, but dnf cannot see it. I'm using Rocky 9.3

```
 uname -r
5.14.0-362.18.1.el9_3.x86_64
```

```
[root@DL360G9 linktest]# dnf list available --showduplicates zfs
Last metadata expiration check: 0:00:10 ago on Wed 28 Feb 2024 03:20:28 AM EST.
Available Packages
zfs.x86_64                    2.1.13-2.el9                    zfs
zfs.x86_64                    2.1.14-1.el9                    zfs
zfs.x86_64                    2.1.14-2.el9                    zfs
zfs.x86_64                    2.2.1-1.el9                     zfs-testing
zfs.x86_64                    2.2.2-1.el9                     zfs-testing
zfs.x86_64                    2.2.2-2.el9                     zfs-testing
```

When I look at http://download.zfsonlinux.org/epel-testing/9.3/x86_64/ then I can see 2.2.3 packages that are older than the 2.2.3 GA date.

What can I do to get rpms (except compiling myself).

Thanks,
Hannes

**Comment by tonyhutter** [comment]
@hanneskasp I'm seeing that on my end too.  Maybe the repo files did not get updated?  Let me see if I can fix it.

**Comment by tonyhutter** [comment]
@hanneskasp ok, please give it another try.  The problem was the `epel-testing/9.3/` directory got updated but not the `epel-testing/9/` directory.

**Comment by hanneskasp** [comment]
Hello,
now it works. Thank you for fixing!

zfs --version
zfs-2.2.3-1
zfs-kmod-2.2.3-1

Bes regards,
Hannes
